webui/react/src/components/LogViewer/LogViewer.tsx

```diff
@@ -564,6 +572,26 @@ const LogViewer: React.FC<Props> = ({
     [dateTimeWidth],
   );
 
+  const handleKeyboardScrolling = useCallback((e: KeyboardEvent) => {
+    const active = document.activeElement;
+    const next = active?.nextElementSibling;
+    const prev = active?.previousElementSibling;
+    if (e.key === 'ArrowDown' && next instanceof HTMLElement) {
+      e.preventDefault();
+      next.focus();
+    } else if (e.key === 'ArrowUp' && prev instanceof HTMLElement) {
+      e.preventDefault();
+      prev.focus();
+    }
+  }, []);
+
+  const keyboardScrollingRef: RefCallback<HTMLDivElement> = useCallback(
+    (node) => {
+      node?.addEventListener('keydown', handleKeyboardScrolling, { signal: canceler.signal });
```

Comment:
I think its not recommended to use `addEventListener` in React

Reply:
We use `addEventListener` elsewhere in our codebase. How else would you recommend detecting arrow key presses?

Reply:
Maybe `useRef`?
I think the issue comes from Antd tab component, so we need a versatile solution. LogView actually detects arrow keys if its not in antd tab view, and any views in tab component dont detect arrow keys somehow.